src/scripts/nurses_analyses/nurses_scenario_analyses.py

```diff
@@ -0,0 +1,195 @@
+                    "tclose_overwrite": 1,
+                    "tclose_days_offset_overwrite": 7,
+                },
+                'ImprovedHealthSystemAndCareSeekingScenarioSwitcher': {
+                    'max_healthcare_seeking': [False, False],
+                    'max_healthsystem_function': [False, False],
+                    'year_of_switch': 2025,
+                }
+            },
+        )
+
+    # def _baseline_scenario(self) -> Dict:
+    #     return mix_scenarios(
+    #         self._default_of_all_scenarios(),
+    #         {
+    #             'HealthSystem': {
+    #                 'ResourceFile_HR_scaling_by_level_and_officer_type': "historical_scaling",
+    #                 'mode_appt_constraints_postSwitch': 2,
+    #                 "use_funded_or_actual_staffing": "actual",
+    #             },
+    #         },
+    #     )
+    #
+    # def _improved_staffing_scenario(self) -> Dict:
+    #     return mix_scenarios(
+    #         self._default_of_all_scenarios(),
+    #         {
+    #             'HealthSystem': {
+    #                 'ResourceFile_HR_scaling_by_level_and_officer_type': "historical_scaling",
+    #                 'mode_appt_constraints_postSwitch': 2,
+    #                 "use_funded_or_actual_staffing": "funded_plus",
+    #             },
+    #         },
+    #     )
+    #
+    # def _worst_case_scenario(self) -> Dict:
+    #     return mix_scenarios(
+    #         self._default_of_all_scenarios(),
+    #         {
+    #             'HealthSystem': {
+    #                 'ResourceFile_HR_scaling_by_level_and_officer_type': "historical_scaling",
+    #                 'mode_appt_constraints_postSwitch': 2,
+    #                 "use_funded_or_actual_staffing": "actual",
+    #             },
+    #         },
+    #     )
+    ####################################################################
+    def _get_scenarios(self) -> Dict[str, Dict]:
+        """Return the Dict with values for the parameters that are changed, keyed by a name for the scenario.
+        """
+        return {
+            "Baseline":
+                mix_scenarios(
+                    self._default_of_all_scenarios(),
+                    {
+                        "HealthSystem": {
+                            'ResourceFile_HR_scaling_by_level_and_officer_type': "default",
+                            'mode_appt_constraints_postSwitch': 2,
+                            "use_funded_or_actual_staffing": "actual",
+                        },
+                    }
+                ),
+
+            "Improved Staffing":
+                mix_scenarios(
+                    self._default_of_all_scenarios(),
+                    {
+                        "HealthSystem": {
+                            'ResourceFile_HR_scaling_by_level_and_officer_type': "default",
+                            'mode_appt_constraints_postSwitch': 2,
+                            "use_funded_or_actual_staffing": "funded_plus",
```

Comment:
So in this scenario, if you want to scale up the nursing staff to the establishment level in the year of 2025: we do not use `"use_funded_or_actual_staffing": "funded_plus"`, which will not work; instead, we create a csv file to store the scale-up factor (e.g. established nursing count/2024 nursing count) that will enable such expansion and set `HR_scaling_by_level_and_officer_type_mode` to the csv file name.